admin-dev/themes/new-theme/js/stock-page/components/products/alert.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <tr>
+    <td colspan="7">
+      <div class="alert alert-info" role="alert">
+        <i class="material-icons">info_outline</i>
+        <p class="alert-text">No product matches your search. Try changing search terms.</p>
+      </div>
+    </td>
+  </tr>
+</template>
+
+<style lang="sass?outputStyle=expanded" scoped>
+  @import "~PrestaKit/scss/custom/_variables.scss";
```

Comment:
Same problem as https://github.com/vuejs/vue-loader/issues/328
I will try to resolve that with sass loader in webpack